Blog/templates/Blog/edit_post.html

```diff
@@ -0,0 +1,30 @@
+{% extends 'Blog/base.html' %}
```

Comment:
This file is very similar to home page. Avoid duplication and reuse the template. If you want to change message on button, pass the message from view function. If you want home page to have extra features, include this template within home page.